src/main/cpp/t03_toggle.cpp

```diff
@@ -24,5 +24,41 @@
 using namespace std;
 
 int t03_toggle() {
+	char s;
+	cin >> s;
+	int t; string y = "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'y' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951806)